roles/openshift_facts/tasks/main.yml

```diff
@@ -6,5 +6,8 @@
     - ansible_version | version_compare('1.9.0', 'ne')
     - ansible_version | version_compare('1.9.0.1', 'ne')
 
+- name: Ensure python-netaddr is installed
+  yum: pkg=python-netaddr state=installed
```

Comment:
@detiber opinion on how to handle this? Should we just add a default that coincides with the default portal_net? String manipulation on that value rather than actually performing proper subnet calculation?